llvm/lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -5099,6 +5099,49 @@ Instruction* cs6475_optimizer(Instruction *I) {
   }
   // END JOHN REGEHR
 
+  // BEGIN SINA MAHDIPOUR SARAVANI
+  // Optimization for checking if fifth bit is zero
+  Value *Z = nullptr;
+  Value *E = nullptr, *D = nullptr, *B = nullptr, *A = nullptr;
+  ICmpInst::Predicate Pred;
+  ConstantInt *C02 = nullptr;
+  ConstantInt *C0 = nullptr;
+  ConstantInt *C1 = nullptr;
+  ConstantInt *C4 = nullptr;
+
+  if (
```

Comment:
please reformat stuff like this to follow the LLVM coding guidelines https://llvm.org/docs/CodingStandards.html